src/0.9.0/sdks-api.md

```diff
@@ -31,14 +31,13 @@
 
 ## Connection and authentication
 
-Immudb run on 3323 default port. Here we connecting a client with default options and
-authenticating using default username and password.
-It's possible to modify defaults on immudb server config folder inside `immudb.toml`
+Immudb runs on port 3323 as the default. The code samples below illustrate how to connect your client to the server and authenticate using default options and the default username and password.
```

Comment:
we should keep "immudb" with the lowercase i even after punctuation (it's a matter of branding)